engines/bops_consultees/spec/system/planning_applications_spec.rb

```diff
@@ -4,13 +4,15 @@
 
 RSpec.describe "Planning applications", type: :system do
   let!(:local_authority) { create(:local_authority, :default) }
-  let!(:planning_application) { create(:planning_application, :pre_application, local_authority:) }
+  let!(:planning_application) { create(:planning_application, :pre_application, local_authority:, user:) }
   let!(:consultation) { create(:consultation, planning_application:) }
   let(:consultee) { create(:consultee, consultation:) }
   let(:sgid) { consultee.sgid(expires_in: 1.day, for: "magic_link") }
   let(:reference) { planning_application.reference }
+  let(:user) { create(:user) }
 
   before do
+    consultation.start_deadline
```

Comment:
I think the consultation factory has a `:started` parameter:

```suggestion
  let!(:consultation) { create(:consultation, :started, planning_application:) }
  let(:consultee) { create(:consultee, consultation:) }
  let(:sgid) { consultee.sgid(expires_in: 1.day, for: "magic_link") }
  let(:reference) { planning_application.reference }
  let(:user) { create(:user) }

  before do
```